superset/config.py

```diff
@@ -585,7 +585,7 @@ def _try_json_readsha(filepath: str, length: int) -> Optional[str]:
 }
 
 # Cache for chart form data
-CHART_FORM_DATA_CACHE_CONFIG: CacheConfig = {
+EXPLORE_FORM_DATA_CACHE_CONFIG: CacheConfig = {
```

Comment:
This is a breaking change that needs to be flagged in `UPDATING.md`. Related to this, I've been thinking we should add a config which makes it possible to restrict the cache types that can be used. By restricting to e.g. Redis, we could ensure that the service doesn't start up if the admin has forgotten to add a recently added cache config.

Reply:
Great catch! I'll update the file. I liked the idea of restricting the cache types. I will do it in a separate PR where I'll also improve the cache_manager initialization.